src/sentry/deletions/defaults/group.py

```diff
@@ -24,6 +24,11 @@
 
 logger = logging.getLogger(__name__)
 
+GROUP_CHUNK_SIZE = 1000
+EVENT_CHUNK_SIZE = 10000
```

Comment:
I need to investigate further but we're consistently fetching 10 events at a time rather than 10k.
<img width="339" height="334" alt="Image" src="https://github.com/user-attachments/assets/c73d19c9-d15d-4c33-9e2c-fdf7aa8b98ac" />

Here's the fetching of the events method.

https://github.com/getsentry/sentry/blob/21404f3f747c95a7da9c35fa8800185decadb2ac/src/sentry/deletions/defaults/group.py#L91-L114